### PR TITLE
Remove coverity build

### LINKED ISSUE #1334: Stop trying to run coverity builds
I noticed that the [coverity badge is broken](https://github.com/FakeItEasy/FakeItEasy/blob/develop/README.md)
image

because the `https://scan.coverity.com/projects/fakeiteasy-fakeiteasy` endpoint is gone, and I am not sure how to rectify.

Then I noticed that we hadn't been running coverity builds since [build 539](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/build/539).

An exhaustive search revealed that we'd lost the crontab setting for nightly builds after [AppVeyor removed scheduled builds for free accounts](https://www.appveyor.com/blog/2017/11/08/sheduled-builds-for-free-accounts/). We can request they be reinstated for our account, but I don't think we've once had a meaningful finding from the coverity analysis. And even if we turned the build back on, we'd still have a broken badge!

Me, I'd just drop coverity. I don't think it pays its way.

### PR DESCRIPTION
**Pull request by thomaslevesque**
Fixes #1334

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -6,19 +6,13 @@ pull_requests:
...
     GITHUB_TOKEN:
         secure: 5dFninlYVqNF98Pk9ykQwPU2pBFeQmbMOiFG2iSkg+hhcwF3UfmnDB07lvOwYtez
     NUGET_API_KEY:
         secure: ynOI8xETJ5V28b5ij2huIKbWRvqJU8QtLtFf8FLTJCWbxmcVP/OPpcF4B8jcOOKW
 
 build_script:
-- cmd: |
-    if defined APPVEYOR_SCHEDULED_BUILD (set BUILD_TARGET=coverity)
-    build.cmd %BUILD_TARGET%
+- cmd: build.cmd %BUILD_TARGET%
```
Comment (blairconrad) [inline appveyor.yml]: `BUILD_TARGET` is always empty now. We could remove it.

File: tools/build.csx [hunk 2]
```diff
@@ -88,53 +83,6 @@ targets.Add("build", DependsOn("clean", "restore", "versionInfoFile"), () => Run
...
-
-        var version = ReadCmdOutput(".", gitversion, "/showvariable SemVer");
-        var coverityToken = Environment.GetEnvironmentVariable("COVERITY_TOKEN");
-        var coverityEmail = Environment.GetEnvironmentVariable("COVERITY_EMAIL");
-        var repoCommitId = Environment.GetEnvironmentVariable("APPVEYOR_REPO_COMMIT");
-
-        var coverityZipFile = coverityDirectory + "/coverity.zip";
-        Cmd("7z", $"a -r {coverityZipFile} {coverityResultsDirectory}");
-
-        using (var client = new HttpClient())
```
Comment (blairconrad) [inline tools/build.csx]: Now we don't need the System.Net.Http lines above.

**Comment by blairconrad** [comment]
I'm enjoying the sea of ~~pink~~ deleted code.

File: appveyor.yml (hunk 1 above)
Comment (thomaslevesque) [reply appveyor.yml]: Good point!

File: tools/build.csx (hunk 2 above)
Comment (thomaslevesque) [reply tools/build.csx]: Ah, I was counting on VSCode to tell me this kind of thing, but it didn't... I'll remove the unnecessary usings.

**Comment by thomaslevesque** [comment]
I made a fixup commit.

> I'm enjoying the sea of ~~pink~~ deleted code.

Are you afraid my colorblindness doesn't let me see pink ? :wink:

File: tools/build.csx [hunk 3]
```diff
@@ -3,8 +3,6 @@
 #r "System.Net.Http"
 #r "System.Xml.Linq"
 
-using System.Net.Http;
```
Comment (blairconrad) [inline tools/build.csx]: Maybe remove the `#r`s too?

Comment (blairconrad) [reply tools/build.csx]: Oh, and don't worry about fixups. I can keep up with the changes if you just amend.

**Comment by thomaslevesque** [comment]
> Oh, and don't worry about fixups. I can keep up with the changes if you just amend.

Too late ;)
I'll squash it, then.

File: tools/build.csx (hunk 3 above)
Comment (thomaslevesque) [reply tools/build.csx]: > Maybe remove the #rs too?

Oops. I should have thought of it

**Comment by blairconrad** [comment]
> Are you afraid my colorblindness doesn't let me see pink ? :wink:

Well, I'm at least enjoying making a comment that seems like I'm sensitive to any issues that you might have, but that really just emphasizes them.

I'm sure you can see pink. I'm just not sure you can see it as being different from green (I remember one PR a while back when you and I were having a confusing discussion because you thought I'd added code, not removed it. Or maybe vice versa).

**Comment by blairconrad** [comment]
Very nice. Thanks!

**Comment by thomaslevesque** [comment]
Thanks for the merge, and the review !

**Pull request merged**